docs/simulation/calculations.yaml

```diff
@@ -51,7 +51,44 @@ Economy:
       formula: MGQ * GOLD_TO_CASH_RATE
       MGQ: Money good quanity = total production of goods with money = yes
       GOLD_TO_CASH_RATE: defines.country.GOLD_TO_CASH_RATE
+    Overseas maintenance:
+      See: https://github.com/schombert/Project-Alice/blob/main/docs/rules.md#overseas-penalty
```

Comment:
It would be better for this to have the overseas maintenance explanation copied into text directly here with the link as a reference, rather than only having the link.
Also it's probably better to use a permanent link (`https://github.com/schombert/Project-Alice/blob/b7889a9c302feed851c039f1a98d73b75be3215a/docs/rules.md#overseas-penalty`) so this always refers to the same text, rather than using a link to their `main` branch which could change (e.g. the file gets moved and the link invalidated).